core/java/iesi-core/src/main/java/io/metadew/iesi/component/http/HttpQueryParameter.java

```diff
@@ -1,11 +1,15 @@
 package io.metadew.iesi.component.http;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 
 @Data
+@AllArgsConstructor
+@Builder
 public class HttpQueryParameter {
 
     private final String name;
-    private final String value;
+    private String value;
```

Comment:
why no longer final

Reply:
Because if a parameter defined at action level already exists at component level, the method overrides the value of that parameter (setValue).

Reply:
not if you follow the solution proposed below. Don't forget to refactor back to final when finished